e2e/deployment/src/globalSetup.ts

```diff
@@ -0,0 +1,19 @@
+import { exec as execCB } from 'node:child_process';
+import { writeFileSync } from 'node:fs';
+import { promisify } from 'node:util';
+import { getAllDeployedApps } from 'zephyr-agent';
+import { testTargetsPath } from './constants';
+
+const exec = promisify(execCB);
+
+export default async function globalSetup() {
+  console.log('\n - Global Setup - Setting up test cases\n');
+  const output = await exec('npx nx show projects --t=build --exclude libs/*');
+  const examples = output.stdout.split('\n').filter(Boolean);
+  const deployedApps = await getAllDeployedApps();
```

Comment:
as far as we spoke, build should be run befoeAll with get all deployed apps
meaning this file could and should be removed

Reply:
fixed